packages/format/src/format.ts

```diff
@@ -0,0 +1,70 @@
+
+import * as _ from 'lodash';
+
+import { IConstraintModule, ConstraintValue, ConstraintExecuter, IRuntimeContext, ValidationError, IValidator, ConstraintConfiguration } from '@configurable-validator/core'
+
+export class FormatModule implements IConstraintModule {
+
+    initialize(validator: IValidator) {
+    }
+
+    buildConstraintExecuter(value: ConstraintValue, externalData?: any): ConstraintExecuter {
+        return (data: object, context: IRuntimeContext) => {
+            let result : boolean = true;
+            let validationError : ValidationError;
+
+            if(value === 'string' && !_.isString(data)){
```

Comment:
why not using `typeof`?

Reply:
Lodash use it internally, and add some more edge cases